tests/admin_utils.cpp

```diff
@@ -535,4 +535,9 @@ namespace Admin {
         auto parsed_response = static_cast<bson::BsonDocument>(bson::parse(result.body));
         m_access_token = static_cast<std::string>(parsed_response["access_token"]);
     }
+
+    void Admin::Session::trigger_client_reset(int64_t file_ident) const
+    {
+        auto app = apps[m_app_id]["sync"]["force_reset"].request(app::HttpMethod::put, util::format("{\"file_ident\": %1}", file_ident));
```

Comment:
Contrary to the adminbass.sh (see https://github.com/10gen/baas/blob/2aea639bbe2b3e189df78556b52889da471bdd23/etc/scripts/adminbaas.sh#L125C25-L125C40) it seems to me this is actually exposed through the private API: https://github.com/10gen/baas/blob/2aea639bbe2b3e189df78556b52889da471bdd23/api/private/v1/sync.go#L40